openid-4-verifiable-presentations-1_0.md

```diff
@@ -1900,88 +1900,137 @@ these constraints. The Wallet is not controlled by the Verifier and the Verifier
 `values`. This SHOULD include, in particular:
 
-- ensuring that a Verifier, in the response, cannot distinguish between the case where a user did
-  not consent to releasing the credential and the case where the claim value did
+- ensuring that a Verifier, in the response, cannot distinguish between the case where an End-User did
+  not consent to releasing the Credential and the case where the claim value did
   not match the expected value, and
 - preventing repeated or "silent" requests leaking data to the Verifier without
   the user's consent by ensuring that all requests, even if no response can be
-  sent by the Wallet due to a `values` mismatch, require some form of user
+  sent by the Wallet due to a `values` mismatch, require some form of End-User
   interaction before a response is sent.
 
 In both cases listed here, it needs to be considered that returning an error
 response can also leak information about the processing outcome of `values`.
 
-# Privacy Considerations
+### Strictly Necessary Claims {#strictly_necessary}
 
-Many privacy considerations are specific to the credential format and associated proof type used in any particular presentation.
-This section focuses on privacy considerations that are specific to the presentation protocol with some treatment also
-given to considerations that apply across some common credential formats.
+Verifiers SHOULD design DCQL queries that request only the minimal set of claims and Credentials needed to fulfill the specified purposes.
 
-## Selective Disclosure {#seletive-disclosure}
+## Verifier-to-Verifier Unlinkability
 
-Selective disclosure is a data minimization technique that allows for sharing only the specific information needed from
-a credential without revealing everything.
+Wallet can use ephemeral Credentials only to achieve cross-session unlinkability. Wallet can use different instances of Credentials to different Verifiers to achieve cross-Verifier unlinkability. Considerable discourse regarding unlinkability in salted-hash based selective disclosure mechanisms is provided in section 10.1 of [@!I-D.ietf-oauth-selective-disclosure-jwt]. One technique mentioned to achieve some important unlinkability properties is the use of batch issuance, which is supported in [@!OpenID4VCI], with individual Credentials being presented only once.
 
-The DCQL helps facilitate selective disclosure by allowing the Verifier to specify the claims it is interested in,
-allowing the Wallet to disclose only the claims that are relevant to the Verifier's request.
+## No Fingerprinting of the Wallets {#no_fingerprinting}
+
+Verifier SHOULD NOT attempt to fingerprint the Wallets to track the End-User's visits.
+
+## Information Security {#information_security}
+
+Wallet providers and Verifiers SHOULD choose the options in the protocols described in this document so that the data in transit is integrity, confidentiality and availability protected. 
```

Comment:
```suggestion
Wallet providers and Verifiers SHOULD choose the options in the protocols described in this document so that the data in transit is integrity protected and confidentiality over the data is maintained. 
```

Now that I've made this edit, I'm not sure we should have this statement because regardless of the choices made I would argue integrity and confidentiality is maintained its just based on what security assumptions one is willing to make e.g what are you willing to trust TLS, the browser etc.

Reply:
On the above basis, suggesting we remove this sentence.

```suggestion
```

Reply:
agreed.